### PR TITLE
Feature/responsive styles

### PR DESCRIPTION
**Pull request by OxiBo**

### CODE REVIEW

File: applications/directory/src/components/CreateProfileForm.js [hunk 1]
```diff
@@ -307,6 +307,7 @@ const CreateProfileForm = ({
             '& h3': {
```
Comment (chaddjohnson) [inline applications/directory/src/components/CreateProfileForm.js]: Need to move `&` styles to the respective elements. No more nested styles here. Time to clean this up.

File: applications/directory/src/components/Link.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import NextLink from 'next/link';
 import { Link as MuiLink } from '@mui/material';
 
 const Link = forwardRef(({ href, children, ...props }, ref) => {
-  const isExternal = href.match(/^https?:/);
+  const isExternal = href.match(/^https?:/); // TODO - make url that starts with www. external as well??
```
Comment (chaddjohnson) [inline applications/directory/src/components/Link.js:8]: Nope -- all links should always start with http or https. Links starting with just www are invalid. Good thinking though.

File: applications/directory/src/components/ProfileDetails.js [hunk 3]
```diff
@@ -14,7 +15,7 @@ const displayUrlReg = /http(s)?(:)?(\/\/)?|(\/\/)?(www\.)?/;
 
 const socialIcons = {
   website: <LanguageIcon />,
-  behance: <SvgIcon component={BehanceIcon} />,
+  behance: <SvgIcon component={BehanceIcon} />, // TODO - it does not show the icon
```
Comment (chaddjohnson) [inline applications/directory/src/components/ProfileDetails.js:19]: To fix SVG display, can you try solutions 1 and 2 here, in order? https://stackoverflow.com/a/56418369/83897

Here is an example of modifying webpack loader rules in Next.js: https://github.com/neatowebsolutions/greatupsells/blob/master/applications/shopify-admin/next.config.js#L48

File: applications/directory/src/components/ProfileDetails.js [hunk 4]
```diff
@@ -148,36 +169,54 @@ const ProfileDetails = ({ artist, isLoggedIn = false }) => {
...
+                    color: 'text.secondary',
+                    textTransform: 'capitalize'
+                  }}
+                >
+                  {name}
+                </Box>
+              </Box>
+              <Box
+                sx={{
+                  '& a': {
```
Comment (chaddjohnson) [inline applications/directory/src/components/ProfileDetails.js]: Can we move this `& a` styling to the `Link` component below? To support this, the `Link` component would probably need to change link this:

```javascript
<Box as="a" href={href} target="_blank" rel="noopener noreferrer" {...props}>
  {children}
</Box>
```

File: applications/directory/src/components/ProfileReview.js [hunk 5]
```diff
@@ -91,40 +92,74 @@ const ProfileReview = ({ onSubmit }) => {
...
             </Typography>
-            <RadioGroup
-              row
-              aria-label="Approval status"
-              name="row-radio-buttons-group"
-              value={values.approvalStatus}
+
+            <Box
+              sx={{
+                marginBottom: '0.875rem' // this margin is all over the place on the review page (at least 5 different places)
```
Comment (chaddjohnson) [inline applications/directory/src/components/ProfileReview.js]: Maybe use a variable as we discussed (your idea) where `marginBottom: '0.875rem'` is used?

File: applications/directory/src/components/ProfileReview.js [hunk 6]
```diff
@@ -193,7 +238,7 @@ const ProfileReview = ({ onSubmit }) => {
           >
             <Button
               type="submit"
-              disabled={!isValid || !dirty || isSubmitting}
+              disabled={!isValid || !dirty || isSubmitting} // TODO - delete !isDirty because it disables the button for approve cases
```
Comment (chaddjohnson) [inline applications/directory/src/components/ProfileReview.js]: True. Maybe `!dirty` does not belong here. Good catch.

File: applications/directory/src/components/ProfileReview.js [hunk 7]
```diff
@@ -91,40 +92,74 @@ const ProfileReview = ({ onSubmit }) => {
...
+              </RadioGroup>
+            </Box>
+            {/* <Divider light /> */}
           </Box>
 
           {values.approvalStatus === 'rejected' && (
             <Fade in={values.approvalStatus === 'rejected'}>
               <Box
                 sx={{
                   '& p': {
```
Comment (chaddjohnson) [inline applications/directory/src/components/ProfileReview.js]: Need to relocate this `& p` styling to the appropriate element (if there is one?).

File: applications/directory/src/components/Toolbar.js [hunk 8]
```diff
@@ -15,7 +15,7 @@ function Toolbar() {
           '& li': {
```
Comment (chaddjohnson) [inline applications/directory/src/components/Toolbar.js]: Need to move this styling to the `ListItem` element below.

File: applications/directory/src/components/Toolbar.js [hunk 9]
```diff
@@ -15,7 +15,7 @@ function Toolbar() {
...
             margin: '0.625rem 0',
             width: 'auto',
-            padding: '0.5rem 0',
+            padding: ['0.5rem 1rem', '0.5rem 1.5rem'],
             '& a': {
```
Comment (chaddjohnson) [inline applications/directory/src/components/Toolbar.js]: Need to move this styling to the `Link` element below.

File: applications/directory/src/components/WorkExamples.js [hunk 10]
```diff
@@ -6,16 +7,36 @@ function WorkExamples({ images }) {
...
+          fontWeight: 500,
+          marginBottom: '0.875rem'
+        }}
+      >
+        Work Examples
+      </Typography>
+      <Box
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: [
```
Comment (chaddjohnson) [inline applications/directory/src/components/WorkExamples.js:28]: Nice job using Grid 👍

**Review by chaddjohnson — CHANGES_REQUESTED** [review]
Let me know if there are other things you'd like to discuss about this pull request (specific lines would be best).

File: applications/directory/src/components/Link.js (hunk 2 above)
Comment (OxiBo) [reply applications/directory/src/components/Link.js:8]: got it

Comment (OxiBo) [reply applications/directory/src/components/Link.js:8]: I will take care of it in oksana branch

File: applications/directory/src/components/ProfileDetails.js (hunk 4 above)
Comment (OxiBo) [reply applications/directory/src/components/ProfileDetails.js]: done

File: applications/directory/src/components/ProfileReview.js (hunk 5 above)
Comment (OxiBo) [reply applications/directory/src/components/ProfileReview.js]: done

File: applications/directory/src/components/ProfileReview.js (hunk 7 above)
Comment (OxiBo) [reply applications/directory/src/components/ProfileReview.js]: these are styles for error messages which are not hard coded

File: applications/directory/src/components/ProfileReview.js (hunk 6 above)
Comment (OxiBo) [reply applications/directory/src/components/ProfileReview.js]: will be done in oksana branch

File: applications/directory/src/components/Toolbar.js (hunk 9 above)
Comment (OxiBo) [reply applications/directory/src/components/Toolbar.js]: done

File: applications/directory/src/components/WorkExamples.js (hunk 10 above)
Comment (OxiBo) [reply applications/directory/src/components/WorkExamples.js:28]: I think Jacky started it :) I did not know you like grid

**Comment by chaddjohnson** [comment]
@OxiBo Thank you for all your hard work. You're really doing great work.

I fixed the SVG support; see this commit: https://github.com/neatowebsolutions/artistdirectory/commit/9054c042cb2871aa61191e87ef915ad7d115c949.

Note that in that commit I moved the behance SVG icon file to `src/icons`. When loading svg icons via `import`, we only want to load them from within the `src` directory as SVG icons will be built into the actual .js source. Images/assets in the `public` directory will be loaded separately from the .js file by the browser. This part wasn't necessary to make the SVG display; this was just a minor refactor.

**Pull request merged**